users/stanrc85/stanrc85.h

```diff
@@ -1,12 +1,21 @@
 #pragma once
 
 #include "quantum.h"
+#include "version.h"
 
 #define DEFAULT 0  //Custom ANSI
 #define LAYER1 1   //Default ANSI (enable with Fn2+CAPS)
 #define LAYER2 2   //Function keys, arrows, custom shortcuts, volume control
 #define LAYER3 3   //RGB Underglow controls and RESET
 
+enum custom_keycodes {
+  KC_MAKE = SAFE_RANGE,
+  NEW_SAFE_RANGE  //use "NEW_SAFE_RANGE" for keymap specific codes
+};
+
+#define MODS_SHIFT_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
```

Comment:
haha, I should probably update my code too, but these aren't needed anymore.  `MOD_MASK_*` is a core thing now, but this is fine, for now.

Reply:
Really? I got errors about them before I added here. Maybe I missed an include.